app/views/UnauthorisedWithGroupAccessView.scala.html

```diff
@@ -0,0 +1,37 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.components.Heading
+@import config.FrontendAppConfig
+
+@this(
+        main_template: MainTemplate,
+        heading: Heading,
+        config: FrontendAppConfig
+)
+
+@()(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    title = messages("unauthorisedWithGroupAccess.title"),
+    showBackLink = true
+) {
+    @heading(messages("unauthorisedWithGroupAccess.heading"))
```

Comment:
The heading component already does the messages logic, so this can just be @heading("unauthorisedWithGroupAccess.heading")